app/views/course/assessment/answer/_explanation.html.slim

```diff
@@ -0,0 +1,5 @@
+div.panel class="#{ explanation_panel_class(answer) }"
```

Comment:
I believe in interpolations we do not have spaces between {}, but never mind, can fix in the following PR.